src/Accelerators/Accelerator.hpp

```diff
@@ -51,7 +51,7 @@ class Accelerator {
   enum class Kind {
     // clang-format off
     NONE,
-    APPLY_TO_ACCELERATORS(CREATE_ACCEL_ENUM)
+    APPLY_TO_ACCELERATORS(CREATE_ACCEL_ENUM, dummy)
```

Comment:
I think the `CREATE_ACCEL_ENUM`  macro accepts a single argument:
```
#define CREATE_ACCEL_ENUM(name) name,
```
So this code should fail to compile.